tables/el.ctb

```diff
@@ -1,6 +1,13 @@
-# LibLouis Table for Greek Braille - Common Rules
+# LibLouis table for both modern and ancient Greek braille
 # ελληνικό σύστημα μπράιγ [Greek Braille System]
 # κέντρο εκπαίδευσης και αποκατάστασης τυφλών [Center for Education and Rehabilitation for the Blind]
+#
+# -----------
+#-name: ελληνικό σύστημα μπράιγ [Greek Braille System]
```

Comment:
I would drop the "[Greek Braille system]".

There was already metadata in [el.tbl](https://github.com/liblouis/liblouis/blob/9c4c5e82da26b7fd81aa8a3dd5bc63c8f0842228/tables/el.tbl). It's better if there aren't two tables with the same metadata, so either change the metadata in el.tbl or remove el.tbl and add the metadata to el.ctb.